includes/Sync/Product_Import.php

```diff
@@ -161,7 +161,7 @@ protected function import_products() {
 			}
 
 			// import or update categories related to the products that are being imported
-			$catelog_category_id = $catalog_object->getItemData()->getCategoryId();
+			$catelog_category_id = Category::get_square_category_id( $catalog_object->getItemData() );
```

Comment:
Would be nice to update any misspellings of `catelog` to `catalog` in this file

Reply:
Sorry for this silly mistake 🤦. Fixed.